package.json

```diff
@@ -18,6 +18,7 @@
   "author": "Automattic",
   "devDependencies": {
     "@automattic/eslint-plugin-wpvip": "^0.3.0",
+    "@babel/plugin-syntax-decorators": "^7.18.6",
```

Comment:
Unfortunately, this is required for the `eslint` VS Code extension to work: it appears to load Babel plugins from `node_modules` relative to the project root directory, not relative to `.eslintrc`.